.circleci/ds054_fasttrack_outputs.txt

```diff
@@ -9,12 +9,16 @@ fmriprep/logs/CITATION.tex
 fmriprep/sub-100185
 fmriprep/sub-100185/func
 fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_boldref.nii.gz
+fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-acompcor_mask.nii.gz
 fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-brain_mask.json
 fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-brain_mask.nii.gz
 fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-confounds_timeseries.json
 fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-confounds_timeseries.tsv
+fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-CSF_mask.nii.gz
```

Comment:
```suggestion
fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_label-CSF_desc-acompcor_mask.nii.gz
```

I can't remember where we left the `label-` conversation with masks. But this is something we want to address.

As it is now, this can be read as _nice, and here is my CSF mask_, which is not true. This is an eroded CSF mask, so either we add the label (if BIDS compatible) or we cram it into the `desc` like `desc-acompcorCSF`.